apps/src/lib/levelbuilder/lesson-editor/AddResourceDialog.jsx

```diff
@@ -73,10 +74,10 @@ const initialState = {
   name: '',
   type: '',
   audience: '',
-  pdf: false,
+  include_in_pdf: false,
   assessment: false,
   url: '',
-  downloadUrl: '',
+  download_url: '',
```

Comment:
now i am seeing a mix of snake and camel case for these, and i'm not sure if I am following the strategy here. I will bring up at stand up.

Reply:
Yeah, I'm getting a bit lost in the conventions here